ensemble/volren/volume_data.py

```diff
@@ -75,6 +76,10 @@ class VolumeData(HasStrictTraits):
     # The data as a fortran array
     _raw_data = VolumeArray
 
+    # The filter to apply to the raw data
+    volume_filter = Property(Instance(VolumeFilter))
+    _volume_filter = Instance(VolumeFilter)
```

Comment:
Does this need to be a `Property`? It seems to me that you can get the same functionality with a regular instance trait that runs the filter whenever the trait changes.